doc/samples.md

```diff
@@ -430,4 +430,10 @@ Demo of the usage of WebRTC in Uno WebAssembly. This sample establishes a direct
 
 Port of the XAML Brewer WinUI3 SkiaSharp Sample application ([blog post](https://xamlbrewer.wordpress.com/2023/09/25/getting-started-with-skiasharp-in-winui-3/) and [source code](https://github.com/XamlBrewer/XamlBrewer.WinUI3.SkiaSharp.Sample)).
 
-[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/UI/XamlBrewerUnoApp)
\ No newline at end of file
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/UI/XamlBrewerUnoApp)
+
+### Windowing samples
```

Comment:
@MartinZikmund can you or @morning4coffe-dev help to add a proper structure (ReadMe file, src folder, and doc/assets folder) for the https://github.com/unoplatform/Uno.Samples/tree/master/UI/WindowingSamples, please

Take PongWars sample as an example:
https://github.com/unoplatform/Uno.Samples/tree/master/UI/PongWars

Reply:
Done!